### PR TITLE
Fix test regressions introduced while tests were broken

### PR DESCRIPTION
**Pull request by murrant**
Normally, these fixes would be separate PRs, but tests fail without all fixes.
Each fix is its own commit.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: app/Models/Sensor.php [hunk 1]
```diff
@@ -98,30 +98,34 @@ public static function getIconMap()
...
-            'voltage' => $this->sensor_current * 1.15,
-            'humidity' => 70,
-            'fanspeed' => $this->sensor_current * 1.80,
-            'power_factor' => 1,
-            'signal' => -30,
-            'load' => 80,
-            'airflow', 'snr', 'frequency', 'pressure', 'cooling' => $this->sensor_current * 1.05,
-            default => null,
-        };
+        if ($low) {
```
Comment (Jellyfrog) [inline app/Models/Sensor.php:103]: The current broken test. Wondering if low/high is flipped wrong or if the test data is wrong maybe

Comment (murrant) [reply app/Models/Sensor.php:103]: So, the legacy code had this thing where it flips high and low if one is higher than the other. The new code changes WHEN that happens...
But, to me this test looks like the 0s should be null (unset) which would prevent the swap.  I was trying to figure out the correct setting, but my sleepy brain couldn't reason through it :)

**Comment by Jellyfrog** [comment]
You have opened a can of worms

**Comment by murrant** [comment]
> You have opened a can of worms

I believe I said kicked a hornets nest :)

**Pull request merged**